### ISSUE TITLE
Doubts about the effectiveness of code repositories

### ISSUE THREAD

**Issue by dengzuxuan** [HurDuhwan/2025-1_BCI_Project#2]
Hello, are you the author of the article "Improved Domain Adaptation Network Based on Wasserstein Distance for Motor Imagery EEG Classification"? The article mentioned that it was developed using the TensorFlow framework, but your code uses the PyTorch framework and the effect never converges, the testing accuracy has **always been 25%**
